aries/aries_vcx/src/errors/error.rs

```diff
@@ -150,6 +150,9 @@ pub enum AriesVcxErrorKind {
     #[error("Could not parse a value")]
     ParsingError,
 
+    #[error("Unexpected wallet error")]
+    WalletUnexpected,
```

Comment:
Changed to `WalletError` but adding a string would make the type not `Copy`. I think the whole enum would deserve a refactor as there are many variants specific to vdrtools wallet, but it would be better to do it separately.